### DISCUSSION TITLE
FileUpload video no showing

### DISCUSSION THREAD

**Discussion by jcgmalave** [filamentphp/filament#7724] [Help]
i have this FileUpload element:

Forms\Components\FileUpload::make('videos')
                                    ->directory('products/video')
                                    ->visibility('public')
                                    ->acceptedFileTypes(['video/mp4'])
                                    ->hint('Galeria de videos (hasta 5) hasta 5 MB')
                                    ->hintIcon('heroicon-o-video-camera')
                                    ->preserveFilenames()
                                    ->multiple()
                                    ->getUploadedFileNameForStorageUsing(
                                        fn (TemporaryUploadedFile $file): string => (string) str($file->getClientOriginalName())
                                            ->prepend(Str::slug(trim(Auth::user()->brand->name)).'-'),
                                    )
                                    ->label('Video'),
in the edit page the video preview its no showing and it properly saved and stored in the folder.

**Comment by danharrin** [comment]
I don't think we have a Filepond plugin that supports video preview installed

**Reply by jcgmalave** [reply]
in the V 2 i was able to see a preview of the video uploaded

**Reply by danharrin** [reply]
ok maybe I am wrong then, but I don't think we changed anything

**Reply by jcgmalave** [reply]
When a create a new record and i upload an mp4 video the **CreatePage** renders a preview of the video, i can even play it and watch it, when i try to edit the record, the **EditPage** doesnt show the preview. im gonna keep trying and i'll let you know.

Thanks in advanced.

**Reply by vkrms** [reply]
@jcgmalave Any luck with that? I'm stuck as well.

**Reply by vkrms** [reply]
@danharrin I see a video element
```
<div class="filepond--media-preview"><video controls="true" src="blob:http://localhost:3925/{{UUID}}"></video></div>
```
You can even visit the src and there's a zero length "video"
